### PR TITLE
Implement new alert message with espiresAt

### PR DESCRIPTION
**Pull request by Fuzznimp**
## Description

This PR aims at implementing the new version of the alert messages with an expiration date.
I also fixed some type along the way.

## Type of change

- [ ] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [x] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## Checklist:

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

**Comment by Fenntasy** [comment]
Task linked: [CU-ev4rvf [feature] - Implement expiresAt to alert messages](https://app.clickup.com/t/ev4rvf)

File: src/components/page-layout.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { AlertMessage } from "@fwl/web/dist/src/typings/utils";
```
Comment (Oluane) [inline src/components/page-layout.tsx:1]: "@fwl/web/dist/src/typings/utils"

Is this path the right one ? 🤔

Comment (Fuzznimp) [reply src/components/page-layout.tsx:1]: It is. I also was confused by the path. It has to do with the way web exports its type declations. I'll double check thought, thanks for the reminder ;).

Comment (Fuzznimp) [reply src/components/page-layout.tsx:1]: I have found the issue. I'll wait for the next version of web to be published to update the version and the path here.

File: README.adoc [hunk 2]
```diff
@@ -251,62 +259,72 @@ We are also using the `Command Query Responsibility Segregation`(CQRS) pattern t
 - */main.js*: Config file for Storybook.
 - */preview.js*: File used to inject, through decorators, the design-system theme and global style.
 
+=== bin/
+
+- *dynamodb/*: Scripts related to DynamoDB, used in Github Actions.
```
Comment (Oluane) [inline README.adoc:264]: "used in Github Actions" I'm not sure about this one 🤔

File: README.adoc [hunk 3]
```diff
@@ -427,13 +445,16 @@ export const SvgIcon: React.FC = () => {
 
 When you use Figma to export SVG, be sure to export the upper component (i.e. *name 40x40*).
 
+Also remember to add a `<title/>` JSX tag under the `<svg/>` tag for accessibility.
+
 == Database
 
-TODO
+We are using DynamoDB as our persistence layer. It's K/V structure allows fast performances, but you'll need to be aware of some specificities, like being unable to update an existing value.
```
Comment (Oluane) [inline README.adoc:452]:
```suggestion
We are using DynamoDB as our persistence layer. Its K/V structure allows fast performances, but you'll need to be aware of some specificities, like being unable to update an existing value.
```

**Review by Oluane — APPROVED** [review]

**Pull request merged**